### DISCUSSION TITLE
Suggestion: more prominent Marp markdown link

### DISCUSSION THREAD

**Discussion by justinwiley** [marp-team/marp#156] [Ideas] [answered]
First let me say thanks for this fantastic tool.  It's incredibly useful, and it just works.  Congrats, I'm really enjoying it.

One potential improvement: a more prominent link to Marp's version of markdown.   When I go to marp.app, the first two links take me to the tools to install (makes sense), and github (also makes sense).

However, I don't think theres a link to https://marpit.marp.app/markdown, unless I'm missing it.  Googling 'marp markdown' was where I eventually located it, as the second hit down.  Adding a more prominent reference on the main site might help folks onboard more quickly.

**Comment by yhatt** [comment, marked as answer]
https://marpit.marp.app/markdown is a part of [Marpit](https://marpit.marp.app) documentation, _that is a framework for developer_, and does not cover the all of Marp features. In addition, **this documentation is designed for developers whose using Marpit framework directly** so it would be hard to understand for Marp consumers. To add a reference to Marpit documentation to the main site may be helpful, but will bring user's confusions in the most case.

> An important thing for thinking about this is that we are essentially thinking about Marpit and Marp separately. [Marpit's documentation](https://marpit.marp.app) is targeted at _"developers using the framework"_, and we have not assumed reading by Marp consumers, On the other hand, the new guide is targeted to consumers.
>
> _— https://github.com/marp-team/marp/discussions/126#discussioncomment-1025128_

We have already planned re-designed docs for Marp users: [Re-creation of Marp website for the unified docs | Blog | Marp](https://marp.app/blog/re-creation-of-marp-website).

[![](https://user-images.githubusercontent.com/3993388/120155208-8bd05280-c22b-11eb-8468-83bbc9d27835.png)](https://marp.app/docs)

It is still in progress so there is no links in the website, but you can peek the guide at https://marp.app/docs. Track the progress at #126 if you have interested.
